### DISCUSSION TITLE
Adding Instrumata Shortcuts to Quick Access

### DISCUSSION THREAD

**Discussion by AIUser007700** [iappyx/Instrumenta#49] [General]
Hello iappyx,
First, THANKS SO MUCH for such a great tool, I have just found it and it is **amazing**!
I have couple of questions:

1- I am trying to re-arrange the position of instrumata ribbon by clicking the [right button - Customize Ribbon] so it can appear first next to home tab but the whole intrumata ribbon isn't showing like the other tabs. why is that? Is there a possibility to make it appear as a tab? also this isnt allowing some of the shortcuts in instrumata to be added to a customized ribbon that I developed, is there a fix for this? 

![Instrumata in NOT in Ribbon](https://github.com/user-attachments/assets/18d67844-8006-4940-bc45-ec26933d8d92)

2- I dont know why but not all the icons and shortcuts are appearing in my toolbar of instrumata, even though I have a large screen, I am able to access the buttons for example under table but I'd like to have the whole ribbon appear, is there a fix for this?

![Instrumata icons](https://github.com/user-attachments/assets/e9f11e4b-cd62-4101-8db3-dbe277510cff)

**Comment by iappyx** [comment]
Hi, there is not really a fix for it, unless you hack the customui xml of the plugin, and remove the items you don't need. How large is your screen?
